inc/utils/namespace.php

```diff
@@ -109,16 +109,45 @@ function get_elasticsearch_url() : string {
 	return $url;
 }
 
+/**
+ * Retrieve the elasticsearch version.
+ *
+ * Return the version string if found otherwise 'unknown'.
+ *
+ * @return string
+ */
+function get_elasticsearch_version() : string {
```

Comment:
Could we have this set by cloud maybe in a constant too? It's a bit of a pain when we're pinging off requests to elastic search on potentially unrelated page loads.

Reply:
Yeah if we update TAS to set it as an environment variable we could use that if it exists. Can you create an issue on the cloud module for that?

Reply:
Added an issue to the terraform repo instead as that's where we'd get the info from.